src/Data/Integer/DivMod.agda

```diff
@@ -59,7 +59,7 @@ n%d<d n -[1+ d ]    = n%ℕd<d n (ℕ.suc d)
 
 a≡a%ℕn+[a/ℕn]*n : ∀ n d {d≢0} → n ≡ + (n modℕ d) {d≢0} + (n divℕ d) {d≢0} * + d
 a≡a%ℕn+[a/ℕn]*n _ ℕ.zero {()}
-a≡a%ℕn+[a/ℕn]*n (+ n) sd@(ℕ.suc d) = let q = n NDM.div sd; r = n NDM.% sd in begin
```

Comment:
Slightly unrelated: can we not drop these absurd cases now? `a≡a%ℕn+[a/ℕn]*n _ ℕ.zero {()}`

Reply:
Didn't get caught in my original `grep` thanks to the braces.